inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/service/RecommendationServiceImpl.java

```diff
@@ -891,6 +901,16 @@ public Predictions computePredictions(User aUser, Project aProject,
         return predictions;
     }
 
+    private void publishPredictionUpdateEvent(String aUserName, double aRecommenderCount, 
```

Comment:
I guess `aRecommenderCount` should be an `int` in the argument list too and then later be cast to a double for the progress calculation.